modules/k6/examples_test.go

```diff
@@ -0,0 +1,26 @@
+package k6_test
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go/modules/k6"
+)
+
+func ExampleRunContainer() {
+	// runK6Container {
+	ctx := context.Background()
+
+	absPath, err := filepath.Abs("./scripts/test.js")
+	if err != nil {
+		panic(err)
+	}
+
+	container, err := k6.RunContainer(ctx, k6.WithTestScript(absPath))
+	if err != nil {
+		panic(err)
+	}
+	//}
+
+	container.Terminate(ctx)
```

Comment:
Please checkout this an example of how to do it:

https://github.com/testcontainers/testcontainers-go/blob/main/modules/clickhouse/examples_test.go#L34-L39C2

```go
	defer func() {
		if err := clickHouseContainer.Terminate(ctx); err != nil {
			panic(err)
		}
	}()
	// }
```

Let's make sure the defer block is under the codeblock used by our docs